DynamicTablesPkg/Library/Acpi/Common/AcpiFadtLib/AcpiFadtLib.inf

```diff
@@ -24,7 +24,7 @@
   Arm/ArmFadtGenerator.c
 
 [Sources.IA32, Sources.X64]
-  FadtGeneratorNull.c
+  X64/X64FadtGenerator.c

```

Comment:
I am not sure if the same code can be used on IA32, but you would know better.

Reply:
Other than the comment above, this patch series looks good to me.
Please let me know once you have tested the changes. I will then apply the push label.

Reply:
> I am not sure if the same code can be used on IA32, but you would know better.

AMD uses X64 mode only for DXE phase, I kept the IA32 for backward compatibility.
Intel, OEM or IBV might use this mode.